### ISSUE TITLE
offset and count-by are not working as expected.

### ISSUE THREAD

**Issue by brooksc** [Miserlou/Loop#34]
Readme includes this:

```
$ loop 'echo $COUNT' --count-by 2 --offset 10
10
12
14
[ .. ]
```

When I run it on Mac OS X 10.12.6 with loop 0.3.3 installed per brew instructions I get...

```
$ loop 'echo $COUNT' --count-by 2 --offset 10
0
1
2
3
4
5
6
7
8
9
10
11
[..]
```
offset and count-by are not working as expected.

**Comment by Miserlou** [comment]
Yep, that definately looks wrong locally to me as well. There was a very large PR that essentially rewrote the project, but I think it introduced a lot of regressions and made the project harder for me to maintain. I might revert if this fix isn't immediately apparent to me..

**Comment by Miserlou** [comment]
Okay, this is fixed in published 0.3.4. It was as I assumed - I shouldn't ever accept PR from people with a default avatar!!!
